platform/windows/detect.py

```diff
@@ -169,7 +169,7 @@ def get_flags():
 
 	return [
 		('freetype','builtin'), #use builtin freetype
-		('openssl','builtin'), #use builtin openssl
+		#('openssl','builtin'), #use builtin openssl
```

Comment:
Hmm, weird. I swear I have the worse problems with git.

> $ git rebase -i HEAD~2
Cannot rebase: Your index contains uncommitted changes.
Please commit or stash them.